ethereum-types/src/lib.rs

```diff
@@ -11,6 +11,7 @@
 mod hash;
 mod uint;
 
+//TODO we need scale_info::TypeInfo on Bloom (and probably others of these)
```

Comment:
Are we expecting these impls as part of this PR? Or are you just adding the dependency here in advance?

Reply:
Yes, I was planning to do them here. In fact, I've done this one.

I'm not totally sure what all types could/should have the TypeInfo. Basically, I'm adding it to everything that I need to make Frontier compile.